rules/falco_rules.yaml

```diff
@@ -2453,9 +2453,9 @@
   desc: Netcat Program runs inside container that allows remote code execution
   condition: >
     spawned_process and container and
-    ((proc.name = "nc" and (proc.args contains "-e" or proc.args contains "-c")) or
-     (proc.name = "ncat" and (proc.args contains "--sh-exec" or proc.args contains "--exec" or proc.args contains "-e "
-                              or proc.args contains "-c " or proc.args contains "--lua-exec"))
+    ((proc.name = "nc" and (proc.args contains " -e" or proc.args contains " -c")) or
+     (proc.name = "ncat" and (proc.args contains "--sh-exec" or proc.args contains "--exec" or proc.args contains " -e "
```

Comment:
@Carlangueitor can you please also put a space after `-e` and `-c` too? So as to `--sh-exec`, `--exec` and `--lua-exec`